### ISSUE TITLE
[Adaptive Cards - Home Page] - Underline or chevron is not permanently defined for all the links present on the 'Adaptive Cards' home page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9300]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Tool: Developer Tool

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to all the links present on the main section of the page using tab key.
3. Press F12 to open the developer tool and inspect the code.
4. Verify that, underline or chevron is permanently defined for all the links present on the 'Adaptive Cards' home page or not.

**Actual Result:**
Underline or chevron is not permanently defined for all the links present on the 'Adaptive Cards' home page.

**Note:**
Same issue replicates for all the links present on the 'Schema Explorer', 'Samples' and 'Blog' page.

**Expected Result:**
Underline or chevron should be permanently defined for all the links present on the 'Adaptive Cards' home page to identify it as link.

**User Experience:**
Cognitive limitation users are unable to differentiate between the normal static text and link text while the links present on the page are not highlighted with underline or chevron.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/db3e60ca-f34a-47ec-ad94-6615685d8c8b)

### Card JSON

```JSON
NA
```
